adr/ADR-30.md

```diff
@@ -48,7 +50,10 @@ For a valid input subject:
   a `*` wildcard-token in the Source filter) and `x` (x's instance of a `*` wildcard-token in the Source filter).
 * Source-matching `>` wildcard (multi token) tokens are mapped to the respective `>` position in the Destination format
 * Literal tokens in the Destination format are mapped to the output subject unchanged (position and value)
-* Destinations must make use of _all_ of the wildcard-tokens present in the transform's Source.
+
+### Using all the wildcard-tokens in the transform's Source
+* For transforms that are defined in inter-account imports (streams and services) the destinations _MUST_ make use of _ALL_ of the wildcard-tokens present in the transform's Source.
+* However, for transforms used any other place (i.e. Core NATS account mappings, Subject transforms in streams, stream imports and stream republishing) it is allowed to drop any number of wildcard-tokens.
```

Comment:
Should we say from what version this is true ?

Reply:
Sure, added it